src/reports/advanced-reporting.md

```diff
@@ -2,7 +2,7 @@
 title: Advanced Reporting
 ---
 
-Advanced Reporting gives you access to a suite of dynamic reports that are based on your product, order, and customer data, with a personalized dashboard that is tailored to your business needs. Advanced Reporting is a free, cloud-based service that is powered by [Magento Business Intelligence][1].
+Advanced Reporting, which is free for Magento Commerce and Open Source customers, gives you access to a suite of dynamic reports that are based on your product, order, and customer data, with a personalized dashboard that is tailored to your business needs. While Advanced Reporting uses Magento Business Intelligence (MBI) for analytics, you do not need to have an MBI account to use Advanced Reporting.
```

Comment:
maybe use "included with" instead of "free"?

Reply:
I agree with that suggestion